README.md

```diff
@@ -39,15 +39,15 @@ A youtube gameplay video can be found here: https://www.youtube.com/watch?v=-PEK
 * Changes in UI
   * We have added an instructions scene and a stunning menu with automatic gameplay on the background
 * Changes in logic
-  * We have fixed uncountable bugs which made the game unplayable
+  * We have fixed a very countable number of bugs
 
 <br></br>
-In short, we have turned a quite buggy game into a good looking, bug-free and addictive piece of art.
+In short, we have turned a quite buggy game into a good looking, bug-free and addictive piece of art while destroying the core game design that made Chromacore interesting.  The original authors released an updated version post-GitHub Game Off, [here](https://github.com/Murkantilism/Chromacore), preserving what made the game fun to begin with.
 
 ## The Team
-The project has been continued by a two-member motivated team:
+The project was briefly continued by a two-member motivated team:
```

Comment:
I don't really see the use of 'briefly' since the reason why we continued it was for the competition (GGO15). This repo was built with a clear scope, it was our entry in the contest.